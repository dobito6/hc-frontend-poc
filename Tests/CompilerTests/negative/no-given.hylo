trait P {}
trait Q {}

struct A {}
struct B {}

fun f<where A: Q>() {}

fun g() {
  given B: P {}
  given <where A: P> A: Q {}
  f()
}
