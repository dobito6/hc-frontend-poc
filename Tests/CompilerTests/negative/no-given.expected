negative/no-given.hylo:12.3-4: error: no given instance of 'Q<A>' in this scope
  f()
  ^
